src/Microsoft.ML.SearchSpace/Schema/SchemaBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.ML.SearchSpace.Schema
+{
+    public abstract class SchemaBase
```

Comment:
>public abstract class SchemaBase [](http://example.com/codeflow?start=4&length=32)

I feel like you should combine this file and the `SchemaType.cs` file. The other file is just an enum that is used directly in here.

Reply:
### Resolved